models/categories.model.js

```diff
@@ -12,4 +12,8 @@ const fetchCategories = () => {
     })
     }
 
-module.exports = {fetchCategories};
\ No newline at end of file
+exports.fetchDeletedComment = (id) => {
+    return db.query("DELETE FROM comments WHERE comment_id = $1 RETURNING *",[id]).then(({rows})=>{
```

Comment:
can take a look at rowCount rather than rows. When It's 0, return Promise.reject